ts/screens/wallet/ConfirmToProceedTransactionScreen.tsx

```diff
@@ -0,0 +1,153 @@
+/**
+ * This screen ask the authorization to proceed with the transaction.
+ * TODO: integrate credit card componet for visualization of the payment method selected for the transaction
+ */
+
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  H1,
+  Icon,
+  Left,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { Col, Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { WalletStyles } from "../../components/styles/wallet";
+import AppHeader from "../../components/ui/AppHeader";
+import PaymentBannerComponent from "../../components/wallet/PaymentBannerComponent";
+import I18n from "../../i18n";
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+const styles = StyleSheet.create({
+  space: {
+    justifyContent: "space-between"
+  }
+});
+
+export class ConfirmToProceedTransactionScreen extends React.Component<
+  Props,
+  never
+> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  private goBack() {
+    this.props.navigation.goBack();
+  }
+
+  public render(): React.ReactNode {
+    const IMPORTO = "€ 199,00";
```

Comment:
define a type -> create a mock value outside this component + link to the story that will implement the real data-> pass the mock value to the props of this component